api/resources_portal/test/end_to_end/test_organization_with_two_users.py

```diff
@@ -118,9 +118,10 @@ def test_organization_with_two_users(self, mock_auth_request, mock_record_reques
         # Prof removes all notification settings
         self.client.force_authenticate(user=prof)
 
-        settings_url = reverse(
-            "organization-user-setting-detail", args=[prof.organization_settings.first().id]
-        )
+        # Temporary, I want to submit another PR to make this endpoint accept a user and an org. It makes more sense
```

Comment:
Do you have an issue to track this?

Reply:
I do now (: https://app.zenhub.com/workspaces/refinebio-5c92300a9d1bd30b8d3101f2/issues/alexslemonade/resources-portal/316